<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>476</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="generateGraphButton">
            <property name="text">
             <string>Generate connected graph</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dijaskraButton">
            <property name="text">
             <string>Dijkstra's algorithm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="matrixButton">
            <property name="text">
             <string>Generate distance matrix</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="centreButton">
            <property name="text">
             <string>Compute centre of graph</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="primButton">
            <property name="text">
             <string>Minimal spanning tree (Prim)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="kruskalButton">
            <property name="text">
             <string>Minimal spanning tree (Kruskal)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>141</width>
               <height>231</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QTextEdit" name="textEdit">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>60</y>
                <width>104</width>
                <height>21</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="geometry">
               <rect>
                <x>16</x>
                <y>0</y>
                <width>111</width>
                <height>51</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="text">
               <string>Enter id of vertex for Dijaskra Algorithm</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QPushButton" name="chooseDijaskraButton">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>90</y>
                <width>81</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Confirm</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>130</y>
                <width>121</width>
                <height>51</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="text">
               <string>After finishing Dijkstra's algorithm</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QPushButton" name="shortestPathButton">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>190</y>
                <width>111</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Show shortest paths</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
